test/unit/models/logs.test.js

```diff
@@ -0,0 +1,40 @@
+const chai = require("chai");
+const { expect } = chai;
+const sinon = require("sinon");
+
+const logsQuery = require("../../../models/logs");
+const addUser = require("../../utils/addUser");
+const cleanDb = require("../../utils/cleanDb");
+
+const logData = require("../../fixtures/logs/cacheLogs")();
+const userData = require("../../fixtures/user/user")();
+
+let userId;
+
+describe("Logs", function () {
+  beforeEach(async function () {
+    userId = await addUser(userData[0]);
+  });
+
+  after(async function () {
+    await cleanDb();
+  });
+
+  describe("fetchMemberCacheLogs", function () {
+    it("Should fetch the member cache logs", async function () {
+      sinon.stub(logsQuery, "fetchMemberCacheLogs").returns(logData[0]);
```

Comment:
Why are we stubing this? I don't think there is need to stub any function for fetching cache logs.

Reply:
Stubbing is needed because not every time the API will return the response with logs because we are fetching logs for the specific user and of the last 24 hours. So, in that way the test will be flaky as if it  gets the response it will pass but other times it will fail.

Reply:
That is correct but we have firestore emulator run locally when running test, which means there will be no external API call. So technically all firestore functions are stubbed in tests.

Reply:
I get it but how do we get that data for which we need to write tests. Like there should be some data present while testing this model function. 
This model function doesn't return any data actually in most cases.

Reply:
There is add logs model function. Using that add your mock data.

Reply:
How are you planning on adding cache logs and where?